apply/templates/apply/apply.html

```diff
@@ -40,13 +40,56 @@ <h1 class="mb-5"> Apply </h1>
             <h5 class="card-title">{{ job.business_title }}</h5>
             <p class="card-text text-muted h6"> # {{job.id }} | <a href="{% url 'department_details:department_detail' job.department.id %}">{{ job.department.name }}</a> </p>
             <hr class="h-bar">
-            <p class="card-text"><small>{{job.job_description|truncatechars:500 }} <a href="{% url 'jobs:job_detail' job.id %}">View job></a></small></p>
+            <p class="card-text"><small>{{job.job_description|truncatechars:500 }} <a href="{% url 'jobs:job_detail' job.id %}">View job</a></small></p>
         <br>
 
         <form class="reg_form" method="post" enctype="multipart/form-data">
           {% csrf_token %}
             {{ application|crispy }}
               <button type="submit" class="btn btn-primary"> Submit Application</button>
+                <script type="text/javascript">
+
+                    function checkUpload1(size)
+                    {
+                        if(size>2)
+                        {
+                            var n = size.toFixed(2);
+                            alert('Your file size is: ' + n + "MiB, and it is too large to upload! Please try to upload smaller file (2MiB or less).");
+                            <!--If the resume submission is too large, make the value NULL-->
+                            $('#id_resume').val('');
+                        }
+                        else
+                        {
+                            //alert('File size is OK');
+                        }
+                    }
+
+                    function checkUpload2(size)
+                    {
+                        if(size>2)
+                        {
+                            var n = size.toFixed(2);
+                            alert('Your file size is: ' + n + "MiB, and it is too large to upload! Please try to upload smaller file (2MiB or less).");
+                             <!--If the cover letter submission is too large, make the value NULL-->
+                            $('#id_cover_letter').val('');
+                        }
+                        else
+                        {
+                            //alert('File size is OK');
+                        }
+                    }
+
+                    $('#id_resume').change(function() {
+                    var fileSize = this.files[0].size/1024/1024;
+                        checkUpload1(fileSize);
+
+                    });
+
+                    $('#id_cover_letter').change(function() {
+                    var fileSize = this.files[0].size/1024/1024;
```

Comment:
@ahanif19 what is the purpose of dividing by 1024 and then 1024 again?

Reply:
The file size is returned in bytes. To convert bytes to mebibytes, you have to divide by 1024*1024